src/pages/index.js

```diff
@@ -18,31 +20,103 @@ import {
   addNewCardButton,
   cardList,
   previewImageModal,
+  changeProfilePictureSelector,
+  profileImageButton,
+  currentProfileImage,
+  deleteCardModalSelector,
 } from "../utils/constants.js";
+//import { data } from "autoprefixer";
 
-//Form Validator
-
+//Class Instances
+const deleteCardFormValidator = new FormValidator(
+  config,
+  deleteCardModalSelector
```

Comment:
`deleteCardFormValidator` is not needed because it has no inputs (it's a confirmation popup's form)

Reply:
Removed.